server/server.go

```diff
@@ -353,6 +353,30 @@ func (server HTTPServer) serveAPISpecFile(writer http.ResponseWriter, request *h
 	http.ServeFile(writer, request, absPath)
 }
 
+// addCORSHeaders - middleware for adding headers that should be in any response
+func (server *HTTPServer) addCORSHeaders(nextHandler http.Handler) http.Handler {
+	return http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Access-Control-Allow-Origin", "*")
+			w.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
```

Comment:
is it added for all endpoints? including ones that are only GET or DELETE, for example?

Reply:
Yes

Reply:
why?

Reply:
Because it doesn't make any impact to other routes, but if we want to use it only for some single routes that has `OPTIONS` method, then it means we need to make some wrappers, which are I think make whole logic slightly more complicated.

Reply:
ok get it, let's leave it in current state.